.python-version

```diff
@@ -0,0 +1 @@
+3.10
```

Comment:
3.12

Reply:
Do you think it's better to add the patch as well as the major and minor?

Reply:
not really, but once we pull in from main we will need to. Main is already 3.12